File: AreaOfTriangle.java
package Codes;

import java.util.Scanner;

public class AreaOfTriangle {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        double base,hieght,area;
        System.out.print("Enter Base: ");
        base = input.nextDouble();
        System.out.print("Enter Hieght: ");
        hieght = input.nextDouble();

        area = 0.5 * base * hieght;

        System.out.print("Area = "+area);


    }
}

File: area of triangle.c
#include<stdio.h>
double triangleArea(double base, double hieght);
int main()
{
    double base,hieght;
    scanf("%d %d",&base,&hieght);


    double area = triangleArea(base,hieght);
    printf("%.lf",area);


}
double triangleArea(double base, double hieght)
{
return 0.5*base*hieght;
}
